### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by aberonni**

### CODE REVIEW

File: src/database.types.ts
Comment (aberonni) [inline src/database.types.ts:10]: Another thing I'm confused by is that I have differences in this file without really touching anything in Supabase. Maybe something is off in our configuration but I'm not sure.

**Review by aberonni — COMMENTED** [review]
@rlmead I don't think this branch is ready yet but you can test it on the preview link if you like. I'm commenting on a couple of things also with regard to the Supabase email I sent you.

File: src/app/(root)/auth/profile/page.tsx
Comment (aberonni) [reply src/app/(root)/auth/profile/page.tsx:25]: For now, I'm going to ignore this confusion because it isn't getting in the way of merging this.

File: src/database.types.ts
Comment (aberonni) [reply src/database.types.ts:10]: Resolved in #36

**Comment by aberonni** [comment]
@rlmead I think this one is also good to go, to be merged after the others. Plenty more can be done but I suppose it's worth reviewing this given that it works and can be shipped, before I add more features and make this PR grow even more 😄

**Comment by rlmead** [comment]
> @rlmead I think this one is also good to go, to be merged after the others. Plenty more can be done but I suppose it's worth reviewing this given that it works and can be shipped, before I add more features and make this PR grow even more 😄

Oh nooo I've created conflicts by renaming the GitHub workflow files 🙈 I'll wait for your feedback on that renaming before I do any conflict resolution here, just in case you object to my changes! If you're ok with them, feel free to merge #36 and then we can go from there.

**Comment by aberonni** [comment]
@rlmead there aren't any conflicts actually, it's just Github being apprehensive :D

I've rebased the branch on dev after merging the other branch and there were no conflicts. All good!

**Review by rlmead — APPROVED** [review]
I love this all. 👏🏼

**Pull request merged**
